distributions/verified/manifest.yaml

```
@@ -0,0 +1,73 @@
+dist:
+  module: github.com/solarwinds/solarwinds-otel-collector/verified
+  name: solarwinds-otel-collector-verified
+  description: SolarWinds OpenTelemetry Collector - Verified
+  version: 0.123.7
+  output_path: ./_build
+
+extensions:
+  - gomod:	go.opentelemetry.io/collector/extension/memorylimiterextension v0.123.0
```

```suggestion
  - gomod: go.opentelemetry.io/collector/extension/memorylimiterextension v0.123.0
```
For all entries. If intended, please unify all files.

---

Fixed